official/resnet/keras/keras_imagenet_benchmark.py

```diff
@@ -568,6 +568,26 @@ def benchmark_xla_8_gpu_fp16_tweaked(self):
     FLAGS.data_delay_prefetch = True
     self._run_and_report_benchmark()
 
+  def benchmark_xla_8_gpu_fp16_tweaked_get_next_as_optional(self):
```

Comment:
Try to shorten this, it makes the UI a mess the longer the name gets.    `benchmark_xla_8_gpu_fp16_tweaked_partial_batch`  or something if you can.

Reply:
Done. Renamed to benchmark_xla_8_gpu_fp16_optional_get_next.